jobserver/api/releases.py

```diff
@@ -576,13 +577,22 @@ class Level4AuthenticatedUser(serializers.Serializer):
 
 
 def build_level4_user(user):
+    ongoing_project_statuses = [
+        Project.Statuses.ONGOING,
+        Project.Statuses.ONGOING_LINKED,
+    ]
```

Comment:
Making this a set, rather than a list, would be more consistent with the later test for containment.